src/store/modules/assets/modules/ercNft.ts

```diff
@@ -38,31 +37,10 @@ const ercNft_module: Module<ERCNftModuleState, RootState> = {
             }
         },
         saveCustomContracts(state) {
-            let tokenRawData = state.ercNftTokensCustom
-                .filter((token) => token.data.ercTokenIds.length > 0)
-                .map((token) => token.data)
+            // ercTokenIds will load later in 'updateWalletBalance'
```

Comment:
We'll rollback the code and just modify to make display work, and the followings are situations we encountered and some concerns to discuss
- I tested on the Fuji testnet because there are tens of millions of blocks. I encountered the error `requested too many blocks from 0 to xxxxxxxx, maximum is set to 2048` when calling `getPastLogs`, and I guess we also have the same default setting. (But seems I'm wrong, the `defaultMaxBlocksPerRequest` is set to `0` in our code)
- Since the scanned result stores in `localStorage` now, if the user accesses with the new device, the logs scanning will start from block 0 again, just worry that will cause extra resource consumption to filter the logs in backend if more blocks have been produced and more new coming devices are connected.
- And at last meeting with Alex, he suggest to use indexed API like magellan or something like external wallet cache server, maybe it might be also a good solution